dashboard/app/controllers/api/v1/sections_controller.rb

```diff
@@ -109,6 +109,14 @@ def membership
     render json: @sections, each_serializer: Api::V1::SectionNameAndIdSerializer
   end
 
+  # GET /api/v1/sections/valid_scripts
+  def valid_scripts
+    return head :forbidden unless current_user
+
+    scripts = Script.valid_scripts(current_user).map(&:assignable_info)
+    render json: {scripts: scripts}
+  end
```

Comment:
this method doesn't appear to have anything to do with sections. We should consider eventually moving this to a new /api/v1/scripts_controller. However I'm fine with waiting until after teacher dashboard is moved over to dashboard / rails so that you don't have to fiddle with the rails routes to get this new api appearing in both /api/v1 and /dashboardapi .

Reply:
agreed! i'll start scoping out some better places for it to live and add a ticket in axosoft to revisit it as follow-up work